src/LiveDevelopment/Documents/HTMLDocument.js

```diff
@@ -263,7 +263,8 @@ define(function HTMLDocumentModule(require, exports, module) {
                 console.warn("Browser DOM does not match after change: " + JSON.stringify(change));
 
                 edits.forEach(function (delta) {
```

Comment:
On second thought, wrap the whole thing in an `if(this._debug) {...` so it only happens if we are debugging.